tests/suite/cli_self_upd.rs

```diff
@@ -304,7 +304,24 @@ info: downloading self-update
             &["rustup", "self", "update"],
             &format!("  rustup updated - {version} (from {version})\n\n",),
```

Comment:
This is a bit weird since `version` appeared 2 times, and technically my addition can be a false negative if the `DisplayVersion` is somehow not updated. However I cannot think of a better place to add it... We could possibly change the `DisplayVersion` before `rustup self update` to be sure, but that breaks the sandboxing.

Reply:
I tried removing the `DisplayVersion` and see if `rustup self update` re-adds it. This also breaks the sandbox but in a more harmless way, I expect.